public/video-ui/src/services/WorkflowApi.js

```diff
@@ -0,0 +1,62 @@
+import { pandaReqwest } from './pandaReqwest';
+import { getStore } from '../util/storeAccessor';
+
+export default class WorkflowApi {
+  static get workflowUrl() {
+    return getStore().getState().config.workflowUrl;
+  }
+
+  static workflowItemLink() {
+    //TODO link to atom in Workflow
+    return `${WorkflowApi.workflowUrl}/dashboard?atom-type=media`;
+  }
+
+  static getSections() {
+    return pandaReqwest({
+      url: `${WorkflowApi.workflowUrl}/api/sections`,
+      crossOrigin: true,
+      withCredentials: true
+    }).then(response => {
+      return response.data
+        .map(section => Object.assign({}, section, { title: section.name }))
+        .sort((a, b) => {
+          if (a.title.toLowerCase() < b.title.toLowerCase()) return -1;
+          if (a.title.toLowerCase() > b.title.toLowerCase()) return 1;
+          return 0;
```

Comment:
The [lodash syntax](https://lodash.com/docs/4.17.4#sortBy) looks largely similar to this, I'm tempted to stay with the native `.sort` function. What do you think?